shamir_mnemonic/__init__.py

```diff
@@ -691,9 +713,22 @@ def combine_mnemonics(mnemonics, passphrase=b""):
         for group_index, group in groups.items()
     ]
 
-    return _decrypt(
-        _recover_secret(group_threshold, group_shares),
-        passphrase,
-        iteration_exponent,
-        identifier,
-    )
+    encrypted_master_secret = _recover_secret(group_threshold, group_shares)
+
+    return identifier, iteration_exponent, encrypted_master_secret
+
+
+def recover_mnemonics(mnemonics, passphrase=b""):
+    """
+    Combines mnemonic shares and decrypts the resulting master secret with the provided
+    passphrase.
+
+    :param mnemonics: List of mnemonics.
+    :type mnemonics: List of byte arrays.
+    :param passphrase: The passphrase used to encrypt the master secret.
+    :type passphrase: Array of bytes.
+    :return: The master secret.
+    :rtype: Array of bytes.
+    """
+    mnemonic_data = combine_mnemonics(mnemonics)
+    return decrypt(*mnemonic_data, passphrase)
```

Comment:
Similar to the comment above. `combine_mnemonics` returns EMS, `recover_mnemonics` returns MS.

Reply:
Again agreed, plus recover_mnemonics does not recover mnemonics, it recovers the master secret from the mnemonics.